server/src/termdb.server.init.js

```diff
@@ -106,33 +112,45 @@ export function server_init_db_queries(ds) {
 			return undefined
 		}
 	}
-
+	ds.term2SampleType = new Map()
```

Comment:
to be consistent with nearby code, should attach `term2SampleType` not directly on `ds`, but maybe `ds.cohort.termdb`?